From 4f1c5c5dcdfababaa4cc8990c9ecd5f5940da9a7 Mon Sep 17 00:00:00 2001
From: Robert Richter <rrichter@cavium.com>
Date: Fri, 18 Jul 2014 17:51:06 +0200
Subject: [PATCH 417/430] arm64, defconfig: Enable Cavium Thunder SoC in
 defconfig

This commit comes from:
  git://git.kernel.org/pub/scm/linux/kernel/git/rric/linux.git

This patch enables Thunder SoCs in the arm64 defconfig. This is
esp. useful to add Thunder platforms to automated builds based on
arm64 defconfig.

Signed-off-by: Robert Richter <rrichter@cavium.com>
(cherry picked from commit 7050d9ed1dd0b0c975a31ebc1d8e4f4b36039a32)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/configs/defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index d92ef3c..9cd37de 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -32,6 +32,7 @@ CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 # CONFIG_BLK_DEV_BSG is not set
 # CONFIG_IOSCHED_DEADLINE is not set
+CONFIG_ARCH_THUNDER=y
 CONFIG_ARCH_VEXPRESS=y
 CONFIG_ARCH_XGENE=y
 CONFIG_SMP=y
-- 
1.7.5.4

